tests/static/js/jqunit/jqUnit-binder-commonEventBindings.js

```diff
@@ -0,0 +1,128 @@
+/* globals fluid, jqUnit */
+(function () {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+
+    // Component to test support for common event bindings
+    fluid.defaults("gpii.tests.binder.commonEventBindings", {
+        gradeNames: ["gpii.tests.binder.base", "gpii.binder.bindMarkupEvents"],
+        bindings: {
+        },
+        selectors: {
+            inputButton: "#input-button",
+            inputKeydown: "#input-button",
+            paragraphClick: ".paragraph-click",
+            multipleEventInputButton: "#input-multiple-events-button"
+        },
+        markupEventBindings: {
+            inputButton: {
+                method: "click",
+                args: ["{that}.handleInputClick"]
+            },
+            inputKeydown: {
+                method: "keydown",
+                args: ["{that}.handleInputClick"]
+            },
+            paragraphClick: {
+                method: "click",
+                args: ["{that}.handleParagraphClick"]
+            },
+            multipleEventInputButton: {
```

Comment:
Whilst we should support this form, we also need to support multiple bindings which don't share a handler, so please add a test which shows support for an array or hash as the RHS here

Reply:
added support for an array of bindings here